src/post/post.entity.ts

```diff
@@ -412,6 +412,7 @@ export class Post extends BaseEntity {
         noteContent: string,
         imageUrl?: URL,
         mentions: Mention[] = [],
+        altText?: string,
```

Comment:
I think your pattern of having `{url: URL, name: string}` fits really nice here, rather than adding an extra parameter